src/lib/wallet/providers.ts

```diff
@@ -18,15 +19,6 @@ export const isMetaMask = (provider: ExternalProvider) => (
   && (
     !(provider as InjectedCoinbaseWalletExtensionProvider).overrideIsMetaMask
   )
-
-  /* not a MetaMask wannabe! */
-  && (
```

Comment:
Removed this because some wallets such as OKX wallet purposely override the metamask ethereum provider and this check prevents the override and would skip the provider altogether. This will then fallback to WalletConnect.

I think we should not do this as it should not be the dApp's call on how the user choose to manage there wallet extension defaults/overrides. If the user wants to override their metamask with another wallet, we should let them.